wdio.conf.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const wdioConf = require('terra-toolkit/lib/wdio/conf');
+const localIP = require('ip');
+const path = require('path');
+
+const staticServerPort = 8080;
+
+const config = {
+  ...wdioConf.config,
+
+  baseUrl: `http://${localIP.address()}:${staticServerPort}`,
+  specs: [
+    path.join(__dirname, 'packages', '**', 'tests', 'specs', '**'),
+  ],
+  staticServerPort,
+  staticServerLog: false,
+  staticServerFolders: [
+    { mount: '/', path: path.join(__dirname, 'packages', 'terra-site', 'dist') },
+  ],
+
+  suites: {
+    badge: [
+      path.join(__dirname, 'packages', 'terra-badge', 'tests', 'specs', '**'),
+    ],
+  },
+
+  seleniumDocker: {
+    enabled: !process.env.TRAVIS,
+    cleanup: false,
+  },
+  // eslint-disable-next-line global-require
```

Comment:
Rather than disabling eslint here can we move the require to the top of the file and then reference resulting const here?

Reply:
yeah this might change anyway pending @emilyrohrbough adding a webpack service